ArduCopter/AP_Arming.cpp

```diff
@@ -934,3 +943,18 @@ bool AP_Arming_Copter::disarm(const AP_Arming::Method method, bool do_disarm_che
 
     return true;
 }
+// check if vertical velocity innovations are less than 1m/s (0.5m/s specifically)
+bool AP_Arming_Copter::vel_innovation_check() const
+{
+    Vector3f vel,pos,mag;
+    float tas,yaw;
+    if (!copter.ahrs.get_innovations(vel, pos, mag, tas, yaw)) {
+        return false;
+    } else {
+        if ( fabsf(vel.z) <= 0.5f) {
+            return true;
+        } else {
```

Comment:
```suggestion
        }
```

and reindent rest etc